samples/modules/lvgl/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Missing a copyright line, also other files have copied values, which should be updated.

Reply:
Was not sure how to handle copying file from other overlays :^)